packages/domain/lib/model/login_params.dart

```diff
@@ -0,0 +1,6 @@
+class LoginParams {
+  final String login;
+  final RegExp validRule;
```

Comment:
I'm passing a regular expression so that the validation rules can be changed and improved simply  in constants. Also, thanks to this, the widget becomes more flexible.

Reply:
let's discuss this class tomorrow

Reply:
discussed